app/models/invite.rb

```diff
@@ -286,6 +289,20 @@ def ensure_max_redemptions_allowed
       end
     end
   end
+
+  def valid_domain
+    return if self.domain.blank?
+
+    self.domain = domain.downcase
```

Comment:
```suggestion
    self.domain = self.domain.downcase
```

Minor but I find this more consistent with the surrounding code.

Reply:
Actually could this just be 

```suggestion
    self.domain.downcase!
```